src/test/java/com/shop/frontend/pages/ProductPage.java

```diff
@@ -14,31 +14,36 @@ public class ProductPage {
     By btn_Shop = By.xpath("/html/body/header/div/div/ul/li[2]/a");
```

Comment:
This should be in TA-68?

Reply:
But how can we go to Product page without clicking this button?

Reply:
driver.get("https://webshop-agil-testautomatiserare.netlify.app/products.html");
in the given step

Reply:
I think its good to use url only for main page and for others we should click on the buttons or links, Because we need to replicate the user behaviour.